lib/server.js

```diff
@@ -27,8 +27,16 @@ function registerGet(parsedEndpoint) {
 }
 
 module.exports = function(callback) {
+  var PORT = callback.port || 5555;
```

Comment:
I called that argument `callback` just for testing, but that's problematic since outside of the test context it's the options object. I'm going to rework this soon anyway, so it's fine for now.